content/en/containers/cluster_agent/admission_controller.md

```diff
@@ -130,11 +130,29 @@ Finally, run the following commands:
 
 ### APM and DogStatsD
 
-To configure DogstatsD clients and APM tracers automatically, inject the environment variables `DD_AGENT_HOST` and `DD_ENTITY_ID` by using one of the following:
+To configure and inject APM tracers libraries and DogstatsD clients automatically:
 
 - Add the label `admission.datadoghq.com/enabled: "true"` to your pod.
 - Configure the Cluster Agent admission controller by setting `mutateUnlabelled` (or `DD_ADMISSION_CONTROLLER_MUTATE_UNLABELLED`, depending on your configuration method) to `true`.
 
+To opt-in your container for library injection, use Pod annotations inside your application's YAML file to specify which language tracers and should be injected, along with their versions.
+
+The annotations are a `key: value` pair in the following format:
+
+```yaml
+admission.datadoghq.com/<language>-tracer.version: <tracer-version>
+```
+
+Adding a this annotation means the tracer library for that language and version is injected into the containerized application. You can add several `<language>-tracer.version` annotations to inject multiple language tracers into one container.
+
+For example to inject the latest Java tracer and Ruby tracer version 1.4.0:
```

Comment:
If we're telling people they can inject multiple, they will want to see how that's done, so I added to the example to show two. Is this a realistic example? injecting both Java and Ruby? If no, can you provide a better example?

Reply:
Also, we should list valid language values, so people don't have to guess if it's `node` or `nodejs`, `dotnet` or `dot-net`, etc.

Reply:
+1 on listing all the available values